### PR TITLE
Integrating Android library build process into main build script

### PR DESCRIPTION
**Pull request by xiam**
This PR removes the old `Makefile` we were using for building the `libgojni.so`. We'll be able to use the main build script now:

```
make android-lib
```

Related issue: https://github.com/getlantern/lantern/issues/2385

### CODE REVIEW

**Comment by xiam** [comment]
Hey @atavism, would you like to help me reviewing and testing this out? This will directly affect your current work flow on the Android side. Thanks!

**Comment by atavism** [comment]
@xiam Sure!

File: src/github.com/getlantern/lantern-android/client/masquerades.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ var defaultTrustedCAs = []*ca{
 var cloudflareMasquerades = []*fronted.Masquerade{
```
Comment (xiam) [inline src/github.com/getlantern/lantern-android/client/masquerades.go:24]: Updating masquerades file too.

**Comment by atavism** [comment]
This is working great for me, @xiam. Going to merge these changes now.

**Pull request merged**